### DISCUSSION TITLE
Please add date of release on SDK download page

### DISCUSSION THREAD

**Discussion by dmunsie** [airsdk/Adobe-Runtime-Support#1869] [General]
https://airsdk.harman.com/download

Please add the date of the release. This will make it visually much easier to comprehend how old/new the release is.

(Current)  The latest version of AIR SDK available is AIR 33.1.1.821. The download and usage of this is subj....

(Suggestion) The latest version of AIR SDK available is AIR 33.1.1.821. (04/28/2022) The download and usage of this is subj....

Thank you for your consideration.

**Comment by ajwfrost** [comment]
I can ask the web app team to update that text, but if you click on the "Release Notes" heading link on the top right of the page, it will show you all the release and their dates..

**Reply by dmunsie** [reply]
Yep, I'm aware of the release notes, I just thought it would be convenient to see the release date right next to the build number. :) Thx for the reply.

**Reply by ajwfrost** [reply]
They seemed to like being able to do an update that was nice and simple... it's updated now :-)
thanks

**Reply by dmunsie** [reply]
Nice! Please tell the team that at least 1 person appreciates the update! ;)

**Comment by dmunsie** [comment]
:) Nice! Tell them at least 1 person appreciates that update very much. :)
